docs/conf.py

```diff
@@ -79,9 +79,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = "0.1.6"
+version = "0.1.8"
```

Comment:
Is it correct that the version bumped from 0.1.6 to 0.1.8, and not 0.1.7?

Reply:
Yes, it's correct. Version should be bumped to 0.1.8 as the version number on the stable documentation was not correct.